src/main/java/duke/gui/MainWindow.java

```diff
@@ -87,10 +82,9 @@ private void handleUserInput() throws URISyntaxException, IOException {
     /**
      * Handle action related to "About" menu item.
      *
-     * @param event Event on "About" menu item.
      */
     @FXML
-    private void handleAboutAction(final ActionEvent event) throws URISyntaxException, IOException {
+    private void handleAboutAction() throws URISyntaxException, IOException {
```

Comment:
I really like the use of abstraction, even when handling exceptions you choose to pass it off to another function, which is quite clean.